workspaces/vscode-python/Dockerfile

```diff
@@ -66,39 +28,15 @@ RUN export arch=`uname -m` \
  && mamba clean --all --yes --quiet \
  && pip cache purge
 
-# Install VS Code extensions and clear the extension cache to reduce image size.
+# Install VS Code extensions.
```

Comment:
Why did you remove the "...and clear the extension cache" part of the comment and script?

Reply:
The removal was done in `vscode-base` and I didn't think that `/home/coder/.local/share/code-server/CachedExtensionVSIXs` needed to be removed again. but my understanding of extensions and the cache is very limited!

Reply:
We want this to be done after each time an extension is installed, as the cache will be re-populated with stuff whenever a new extension is installed.